.env

```diff
@@ -0,0 +1,4 @@
+# Copy this file to `.env` and replace "1000:1000" with
```

Comment:
I'm a little confused by this file and references to `.env.dev` in the README.

Should this file actually be named `.env.example` and have the same instruction? How does `.env.dev` end up getting read?